### PR TITLE
[full-ci] Add search support for shares

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/7043

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-app-files/src/components/FilesList/ResourceTable.vue [hunk 1]
```diff
@@ -388,11 +394,32 @@ export default defineComponent({
...
+
+      for (const shareId of uniqueShareIds) {
+        const { shareInfo } = yield owncloudSdk.shares.getShare(shareId)
+        shares.value.push(buildSharedResource(shareInfo, true))
+      }
+    })
+
     return {
+      shares,
+      loadSharesTask,
```
Comment (JanAckermann) [inline packages/web-app-files/src/components/FilesList/ResourceTable.vue:415]: NONONONONO

**Comment by JanAckermann** [comment]
TODO:
-[ ] Find a better way for origin share name handling
-[ ] Adapt for preview

**Comment by JammingBen** [comment]
I wanted to look at https://github.com/owncloud/web/issues/7347, but I believe this PR deals with the same issue, right?

**Comment by JanAckermann** [comment]
Oh yeah, might possibly fix that as well

**Comment by JanAckermann** [comment]
> I wanted to look at #7347, but I believe this PR deals with the same issue, right?

Not a fix for this particular problem, can be done in a follow up pr

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oCIS</strong> https://drone.owncloud.com/owncloud/web/28023/12/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/28023/tracing/public-link-for-space-alice-2022-9-2-10-21-36.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/28023/tracing/public-link-for-space-brian-2022-9-2-10-22-04.zip 


</p></details>

**Comment by SagarGi** [comment]
@JanAckermann  i will have someone to look into this. Putting this to the sprint-board.

**Comment by grgprarup** [comment]
@amrita-shrestha is looking into it.

**Comment by amrita-shrestha** [comment]
@JanAckermann I am going to bump the latest commit id. The test is passing locally in the latest ocis . Maybe tests are failing because of the old commit id. Let's see that will make the CI green or I will investigate further

**Comment by amrita-shrestha** [comment]
@JanAckermann  The remaining tests will probably pass after these changes pull in ocis [Fix if-match](https://github.com/cs3org/reva/pull/3195)

**Comment by JanAckermann** [comment]
@amrita-shrestha Wonderful! Can you take care and restart the ci, if everything is done?

**Review by kulmann — APPROVED** [review]
Works like a charm 😍 
Tooltips are showing misleading paths, but well, that goes much deeper than just building resource links... so probably nothing for this PR.

**Comment by JanAckermann** [comment]
Yes, recognised this too and was like: uagghhhh

**Comment by amrita-shrestha** [comment]
@JanAckermann sorry I rebase your PR in place of restarting CI

**Pull request merged**